### PR TITLE
Dropdown component refactored from other dropdowns

### PR DESCRIPTION
**Pull request by danehansen**
@btholt :eyeglasses:

### CODE REVIEW

File: src/views/components/SortDropdownOld.jsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+import React from 'react';
```
Comment (ajacksified) [inline src/views/components/SortDropdownOld.jsx]: Should this file be removed?

File: src/views/components/ListingDropdownOld.jsx [hunk 2]
```diff
@@ -0,0 +1,200 @@
+import React from 'react';
```
Comment (ajacksified) [inline src/views/components/ListingDropdownOld.jsx]: Should this file be removed?

**Comment by ajacksified** [comment]
Style nits:

* Needs spaces around `=`s
* Needs a space after `:`s
* Single-line `if`s need braces
* No spaces inside the parens for function arguments

**Comment by ajacksified** [comment]
:nail_care:

File: src/views/components/ListingDropdownOld.jsx (hunk 2 above)
Comment (danehansen) [inline src/views/components/ListingDropdownOld.jsx]: oh yea woops

File: src/views/components/ListingDropdown.jsx [hunk 3]
```diff
@@ -12,36 +11,38 @@ import VoteFactory from '../components/Vote';
...
-    this._open = this._open.bind(this);
-    this._close = this._close.bind(this);
+    this._onOpen=this._onOpen.bind(this);
+    this._id = Math.random();
     this._onRemoteVote = this._onRemoteVote.bind(this);
   }
 
   componentDidMount () {
     this.props.app.on(Vote.REMOTE_VOTE+':'+this.props.listing.id, this._onRemoteVote);
+    this.props.app.on(Dropdown.OPEN + ":" + this._id, this._onOpen);
```
Comment (ajacksified) [inline src/views/components/ListingDropdown.jsx]: Should be single quotes, here on the `off` below

**Comment by ajacksified** [comment]
Looks good after style fixes.

**Comment by danehansen** [comment]
:haircut:

**Pull request merged**